src/v/cluster/controller_log_limiter.h

```diff
@@ -98,10 +99,45 @@ class group_limiter {
         }
     }
 
+    void account_dropped() { _dropped_requests_amount += 1; }
+
+    void setup_public_metrics() {
+        if (config::shard_local_cfg().disable_public_metrics()) {
+            return;
+        }
+
+        namespace sm = ss::metrics;
+        _public_metrics.add_group(
+          "cluster:controller_limit",
+          {sm::make_gauge(
+             ssx::sformat("{}_requests_dropped", _group_name),
+             [this] {
+                 auto dropped = _dropped_requests_amount;
+                 _dropped_requests_amount = 0;
+                 return dropped;
+             },
+             sm::description(ssx::sformat(
+               "Amount of dropped requests since last metric in {} group",
+               _group_name))),
+           sm::make_gauge(
+             ssx::sformat("{}_requests_available_rps", _group_name),
+             [this] { return _throttler.available(); },
+             sm::description(
+               ssx::sformat("Available rps for {} group", _group_name))),
```

Comment:
Can you include a detailed description what the metric is related with i.e. add context so the reader can relay on description to understand what the metric value represents